airflow/__init__.py

```diff
@@ -61,7 +61,8 @@
 
 
 # Perform side-effects unless someone has explicitly opted out before import
-# WARNING: DO NOT USE THIS UNLESS YOU REALLY KNOW WHAT YOU'RE DOING.
+# WARNING: Use with care, `_AIRFLOW__AS_LIBRARY` will prevent proper intialization (configs, syspath, logger...).
+#          It might be needed if you intend to use airflow as third-party package.
```

Comment:
```suggestion
# WARNING: DO NOT USE THIS UNLESS YOU REALLY KNOW WHAT YOU'RE DOING.
# This environment variable prevents proper intialization, and things like
# configs, logging, the ORM, etc. will be broken. It is only useful if you only
# access certain trivial constants and free functions (e.g. `__version__`).
```

Reply:
I'm not sure that the 'DO NOT' warning is still relevant here. The description says it might be useful, so it's a bit inconsistent.

Reply:
*UNLESS YOU REALLY KNOW WHAT YOU'RE DOING*.